website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+C &mdash; [Version Control Menu](#version-control-menu)<br />
+D &mdash; [File Explorer](#file-explorer)<br />
+E &mdash; [Command bar](#command-bar)<br />
+F &mdash; [IDE Status button](#ide-status-button)<br />
+G &mdash; [Create New File button](#create-new-file-button)<br />
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-landing.jpg" width="100%" title="The Cloud IDE landing page"/>
+
+
+#### Git Repository link 
+The Git Repository Link displays the name of the active branch. Clicking this element will hyperlink to your repository, pinned to that same branch.
+
+#### Documentation site button
+The Documentation Site button links to the static dbt Documentation site powered by the most recently generated dbt artifacts in the IDE. This button is clickable once a `dbt docs generate` command is successfully invoked in the Command Bar.  
+
+#### Version Control menu
+The Version Control menu contains all git-related elements in the IDE. The Git Actions Button will dynamically update with the relevant action for the state of your editor, including prompt to pull remote changes, commit and sync when reverted commit changes are in the editor, or create a merge/pull request when appropriate. The dropdown menu on the Git Actions Button allows users to revert all changes, refresh git state, create merge/pull requests, and change branches. Refer to [Version control basics](/docs/collaborate/git/version-control-basics#the-git-button-in-the-cloud-ide) for more info. 
+
+Below the Git Actions Button, all changes to files since the last commit will be listed in the **Changes** section. Clicking on any change will open the Git Diff View of the selected file, allowing the user to see the inline changes.
+
+#### File Explorer 
+The File Explorer shows the filetree of your repository. You can:
+* Click on any file in the filetree to open the file in the File Editor. 
+* Click and drag files between directories to move files. 
+* Right click a file to access the sub-menu options like copy file name, copy as `ref`, rename, delete, and more.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-file-explorer-menu.jpg" width="30%" title="Right click files to access sub-menu options"/>
+
+#### Command bar
+The Command bar is where [dbt commands](/reference/dbt-commands) are invoked in the IDE. Invoking a command will open the Invocation History Drawer to show the associated logs.
+
+#### IDE Status button
+The IDE Status Button reports the current status of the IDE. Anytime there is a status error, or if there is a dbt code error that prevents the project from parsing, this button will turn red and say Error. Clicking on this button brings up the [IDE Status Modal](#IDE-Status-Modal)
+
+
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-options-menu.jpg" width="85%" title="Click on the IDE status button to view the options menu"/>
+
+#### Create New File button
+This button allows the user to create a new file in their project. 
+
+## Editing 
```

Comment:
You should call out saving here, be it autosave or the big manual save button + save shortcuts